MockAndroidSBI/app/src/main/java/io/mosip/mock/sbi/mds/MDServiceActivity.java

```diff
@@ -128,24 +127,27 @@ private void handleIntents() {
                     }
 
                     if (null != discoverRequestDto) {
-                        switch (discoverRequestDto.type.toLowerCase()) {
-                            case "face":
+                        switch (discoverRequestDto.type) {
```

Comment:
Can you please add a comment why this can not be small case?

Reply:
Added the comment and reference to the same